lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module DfE
+  module Analytics
+    # Reschedules to run every 24hours
+    class EntityTableCheckJob < AnalyticsJob
+      WAIT_TIME = Date.tomorrow.midnight
+
+      def perform
+        DfE::Analytics.entities_for_analytics.each do |entity_name|
+          DfE::Analytics.models_for_entity(entity_name).each do |model|
+            entity_table_check_event = DfE::Analytics::Event.new
+                                                            .with_type('entity_table_check')
+                                                            .with_entity_table_name(model.table_name)
+                                                            .with_data(entity_table_check_data(model))
+                                                            .as_json
+            DfE::Analytics::SendEvents.perform_later([entity_table_check_event])
+            Rails.logger.info("Processing data for #{model.table_name} with row count #{model.count}")
+          end
+        end
+      ensure
+        reschedule_job
+      end
+
+      def entity_table_check_data(model)
```

Comment:
@ericaporter are we missing the checksum calculation timestamp here, which is different to the ```occurred_at```? Timestamp including time zone data would be preferable as we don't have a way to guarantee time zones are the same between the application and BigQuery.

Reply:
There could be a difference between the two, so will add in a checksum calculation timestamp

Reply:
Now added